packages/hardhat/contracts/GardensTemplate.sol

```diff
@@ -561,12 +471,6 @@ contract HoneyPotTemplate is BaseTemplate {
             _convictionVoting.CREATE_PROPOSALS_ROLE(),
             _disputableVoting
         );
-        _acl.createPermission(
```

Comment:
Not sure about this, because I don't think it should be a single EOA, they would have a lot of power over the garden then. Although the Garden would be able to change it via a vote so perhaps it's okay.

Reply:
I would prefer not to add it to the template, because I think the pause mechanism should be opt-in, not opt-out. We don't have so much space in the template contract either, so if you don't mind I would just not include it in the template.